spec/decorators/term_with_children_spec.rb

```diff
@@ -24,4 +24,23 @@
       end
     end
   end
+
+  describe "#full_graph" do
+    context "with no children" do
+      it "should be the resource" do
+        expect(subject.full_graph.statements.to_a).to eq subject.statements.to_a
+      end
+    end
+    context "with children" do
+      let(:child) { Term.new(uri+"/1") }
+      let(:child_2) { Term.new(uri+"q/2") }
+      before do
+        child.persist!
+        child_2.persist!
+      end
+      it "should have terms of itself and its children" do
+        expect(subject.full_graph.count).to eq child.count + subject.count
```

Comment:
Is this right?  Seems like the total count should include child_2, too, shoulding it?